lib/api.js

```diff
@@ -309,7 +309,9 @@ API.prototype._doRequest = function(method, url, args, cb) {
     if (res.statusCode != 200) {
       if (res.statusCode == 404) 
         return cb({code: 'NOTFOUND'});
-
+      if (err && err.cors == 'rejected') 
+        return cb({code: 'NOTFOUND'});
```

Comment:
Should we be returning the same error when getting a 404 and when getting a cors error? I think a cors error means the server is unreachable (either because the url is incorrect or because the server is down or headers are set to not allow the connection). But a 404 means the programmer of the client application made a mistake and not the user. So it should be informed as "An internal error ocurred" and properly logged and communicated to the programming team.
What do you think about this?